frontend/src/containers/LearnMoreDialog/LearnMoreDialog.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+    Button,
+    Dialog,
+    DialogContent,
+    DialogDescription,
+    DialogHeader,
+    DialogTitle,
+    DialogTrigger,
+} from '@/components'
+import { DialogTriggerProps } from '@radix-ui/react-dialog'
+import React from 'react'
+
+const LearnMoreDialog = ({
+    onClose,
+}: {
+    onClose?: DialogTriggerProps['onClick']
+}) => {
+    return (
+    <Dialog>
+
+        <DialogTrigger asChild onClick={onClose}>
+        <Button className='text-xs md:text-base'>Learn More</Button>
+        </DialogTrigger>
+
+        <DialogContent className='sm:max-w-[730px]'>
+
+        <DialogHeader>
+            <DialogTitle className='text-2xl'>More on Django India</DialogTitle>
+            <DialogDescription>
+            For the Community by the Community
+            </DialogDescription>
+        </DialogHeader>
+
```

Comment:
Lets not add unnecessary empty lines in between. Can use comments if want to make it readable